umm-spec-lib/src/cmr/umm_spec/validation/platform.clj

```diff
@@ -62,7 +62,8 @@
       ;; The case of 0 or greater than 3 shouldn't exist.  If it is either value then the validation
       ;; software is not working.
       (if-not (<= 1 platform-counts 3)
-        (errors/throw-service-error (Exception. "The platform validation checking for duplicates either has 0 or more than 3 elements, neither is valid. There is a software problem."))
+        ;; TODO: "pull-before-flight" - send this to bamboo and remove comment on success - kondo change
```

Comment:
I'm removing this comment in the next version (I assume changes will be suggested). It is here to mark what I thought was a strange clj-kondo change that I wanted tested in bamboo.